### PR TITLE
[Fonts][Colors][Release] All-In-One!

### PR DESCRIPTION
**Pull request by sampi**
@wiredearp @zdlm @sampi @bomouridsen 

## [Colors]
* Adds `@ts-orange`, `@ts-orange-lite`, `@ts-orange-liter`, `@ts-orange-dark`, `@ts-orange-darker`
  * Supported as a background for Buttons

## [Fonts]
* Adds new versions of Open Sans
* Adds `font-weight: 700;` version of Open Sans
  * Changes `@ts-fontweight-bold: 600;` to `@ts-fontweight-semibold: 600;`
  * Adds new `@ts-fontweight-bold: 700;`

## [Release]
* Uses `yarn` in most places instead of `npm`
* Changes from `grunt release` to `yarn release` and `yarn prerelease`
  * Using https://github.com/webpro/release-it
* Has new `browserstack-runner` script that should make it easier to find errors and not send back false positives!

## [TODO]
* New `ts-icon-comment`
* Updated documentation for the new release commands

### CODE REVIEW

File: tasks/browserstack.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+	);
+	console.log(
+		'                               ░ ░     ░   ▒    ▒ ░  ░ ░    ░░░ ░ ░   ░░   ░    ░       ░'
+	);
+	console.log('                               ░  ░ ░      ░  ░   ░        ░        ░  ░ ░');
+	console.log('');
+};
+const succ = function() {
+	console.log(
+		'                             ██████  █    ██  ▄████▄   ▄████▄  ▓█████   ██████   ██████  ▐██▌'
```
Comment (wiredearp) [inline tasks/browserstack.js]: Perhaps leave a comment that shows how this awesome ascii-art can be changed, if indeed it can be changed

Comment (sampi) [inline tasks/browserstack.js]: Added comments and links as well =)

Comment (wiredearp) [inline tasks/browserstack.js]: 👍

**Comment by wiredearp** [comment]
@dsp Before we release this, remind me that we should investigate why the navigation on Docs is broken just now, at least on iPad (it's of course impossible on this iPad to get a clue about the reason).

**Pull request merged**